test/functional/tests/initialize/test_initialize_load.py

```diff
@@ -0,0 +1,360 @@
+            Fio().create_command() \
+                .io_engine(IoEngine.libaio) \
+                .io_depth(64) \
+                .size(Size(4, Unit.GibiByte)) \
+                .read_write(ReadWrite.randrw) \
+                .block_size(Size(64, Unit.MebiByte)) \
+                .target(f"{core[i].system_path}") \
+                .run()
+
+    with TestRun.step("Stop CAS."):
+        cache.stop()
+        if len(casadm_parser.get_caches()) != 0:
+            TestRun.fail("There are still running caches after stopping service.")
+        if len(casadm_parser.get_cores(cache.cache_id)) != 0:
+            TestRun.fail("There are still running cores after stopping service.")
+
+    with TestRun.step("Load CAS."):
+        cache = casadm.load_cache(cache_dev)
+        caches_count = len(casadm_parser.get_caches())
+        if caches_count != 1:
+            TestRun.fail(f"Expected caches count: 1; Actual caches count: {caches_count}.")
+        cores_count = len(casadm_parser.get_cores(cache.cache_id))
+        if cores_count != core_number:
+            TestRun.fail(f"Expected cores count: {core_number}; Actual cores count: {cores_count}.")
+
+    with TestRun.step("Compare cache configuration before and after load."):
+        if cache_mode != cache.get_cache_mode():
+            TestRun.fail("Cache mode are different.")
+        if cache_line_size != cache.get_cache_line_size():
+            TestRun.fail("Cache line size are different.")
+        if cleaning_policy != cache.get_cleaning_policy():
+            TestRun.fail("Cleaning policy are different.")
+        if cleaning_policy == CleaningPolicy.alru:
+            if alru != cache.get_flush_parameters_alru():
+                TestRun.fail("Cleaning policy parameters are different.")
+        if cleaning_policy == CleaningPolicy.acp:
+            if acp != cache.get_flush_parameters_acp():
+                TestRun.fail("Cleaning policy parameters are different.")
+
+    with TestRun.step("Run FIO again on cache's exported object"):
+        for i in range(0, core_number):
+            Fio().create_command() \
+                .io_engine(IoEngine.libaio) \
+                .io_depth(64) \
+                .size(Size(4, Unit.GibiByte)) \
+                .read_write(ReadWrite.randrw) \
+                .block_size(Size(64, Unit.MebiByte)) \
+                .target(f"{core[i].system_path}") \
+                .run()
+
+    with TestRun.step("Stop caches."):
+        casadm.stop_all_caches()
+
+
+@pytest.mark.parametrize("core_number", [1, 4])
+@pytest.mark.parametrize("cleaning_policy", CleaningPolicy)
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.parametrize("cache_line_size", CacheLineSize)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_load_x_to_one_with_params(cache_mode, cleaning_policy, cache_line_size, core_number):
```

Comment:
Is this test actually somehow different than the previous one?? Looks like copy-paste

Reply:
It's nearly the same, but in this case cache is loaded with maximum parameters given like cache line size, cache mode, cache id. This tests got 150 iterations, so I think it's better to have 2 big tests than 1 huge.